.gitignore

```diff
@@ -1,3 +1,4 @@
 .phpcs-cache
 .phpunit.result.cache
 vendor
+composer.lock
```

Comment:
`composer.lock` should be committed

Reply:
will do.

Reply:
Done in 476fa70, and changed the command in the test from composer update to composer install. But I'm not sure if it is better to have composer.lock. Now this test will only ever test version 1.34.0 of ISAP, instead of testing the last published version.